provision/testdata/flavor_cko_calico-3.23.2.inp.yaml

```diff
@@ -24,8 +24,42 @@ aci_config:
     - mykube_extepg
 
 connectivity_checker:
-
-
+ error_pods_reporting_enable: true
+ reachability_test_enable:
+  periodic_sync:
+   enable_updates: true
+   interval: 120
+  external_url:
+   url: 'google.com'
+  proxy: ""
+ reachability_tests:
+  pod_to_pod:
+    update_status: true
```

Comment:
We dont need update_status field under every test in the input file. We can just capture the "pod_to_pod",  pod_to_clusterIP and so on